go-controller/pkg/ovndbmanager/ovndbmanager.go

```diff
@@ -75,13 +81,14 @@ func ensureOvnDBState(db string, kclient kube.Interface, stopCh <-chan struct{})
 func ensureLocalRaftServerID(db string) {
 	var dbName string
 	var appCtl func(args ...string) (string, string, error)
-
+	clusterStatusRetryCnt := &nbClusterStatusRetryCnt
```

Comment:
I'm a little wary of this because it isn't thread safe. It implies ensureLocalRaftServerID and ensureClusterRaftMembership must be executed serially for the same db or we could have multiple writes/reads at the same time to the int. I think a simple change to using atomic lib counter would be safer.

Reply:
Right now they are executed serially in the same go-routine. If we were to refactor and make them separate go-routines, I agree we will have to handle that by making the counter atomic. Would you like to see that change now? 

How would there be multiple reads/writes at the same time even if they aren't executed serially for the same db? Imagine a sequence: 
`ensureLocalRaftServerID(nb) -> ensureLocalRaftServerID(sb) -> ensureClusterRaftMembership(nb) -> ensureClusterRaftMembership(sb)` 

each call will be separate and read the corresponding retryCnt variable. 

It's only when they are overlapping that we land ourselves into trouble and you rightly pointed that those should then be made atomic, but then we lose the nifty way to reference the retry counters without worrying about writing 
```
if db == nb {
   nbClusterStatusRetryCnt = nbClusterStatusRetryCnt + 1
} else {
  sbClusterStatusRetryCnt = sbClusterStatusRetryCnt + 1
}
``` 
logic at multiple places.